src/auth/roles/index.ts

```diff
@@ -129,6 +129,30 @@ const rolesInitialiser: RolesInitialiser = (client) => {
       const result = await client.raw('SELECT EXISTS ?', [inner]);
       return result.rows[0];
     },
+    getUserRole: async ({ userId, organisationId }) => {
+      const query = await client('user_account_access_role')
+        .select('access_role_id')
+        .where({ user_account_id: userId, organisation_id: organisationId });
+      if (query.length === 0) {
+        throw new Error('User does not exist');
+      }
+      return query[0];
+    },
+    getRolePermissions: async ({ roleId }) => {
```

Comment:
If what we're returning here are actually permissions, then IMO the method should go on the `Permissions` interface.

```js
Permissions.forRole(role)
```